index.src.html

```diff
@@ -1491,11 +1496,16 @@
         is `true`, throw a {{TypeError}}.
     2. If {{VideoEncoder/[[state]]}} is not `"configured"`, throw an
         {{InvalidStateError}}.
-    3. Let |frameClone| hold the result of running the [=Clone VideoFrame=]
+    3. If {{VideoEncoder/[[active orientation]]}} is not `null` and does not match
+        |frame|'s {{VideoFrame/[[rotation]]}} and {{VideoFrame/[[flip]]}} throw a
+        {{DataError}}.
+    4. If {{VideoEncoder/[[active orientation]]}} is `null`, assign |frame|'s
+        {{VideoFrame/[[rotation]]}} and {{VideoFrame/[[flip]]}} to it.
```

Comment:
the last `it` seems a bit ambiguous here, I would explicitly write `{{VideoEncoder/[[active orientation]]}}`